integration/janitor/runner/main.go

```diff
@@ -9,12 +9,15 @@ import (
 )
 
 func main() {
-	if len(os.Args) != 2 {
-		fmt.Println("Expected single namespace name argument")
+	if len(os.Args) != 4 {
+		fmt.Println("Expected namespace name, clusterId, clusterSetId arguments")
 		os.Exit(1)
 	}
 
-	j := janitor.NewDefaultJanitor()
 	nsName := os.Args[1]
+	clusterId := os.Args[2]
+	clusterSetId := os.Args[3]
+
+	j := janitor.NewDefaultJanitor(clusterId, clusterSetId)
 	j.Cleanup(context.TODO(), nsName)
```

Comment:
Not a  blocker but would prefer to use cluster set id in constructor, and cluster id in cleanup call, since this will presumably be used for cleaning up multiple clusters in a multi cluster integration test. We can postpone that change until that test is written however.

Reply:
Yes I do agree. Was planning to make some changes for how the janitor runs for multicluster tests